doc/new-guidelines.md

```diff
@@ -190,6 +190,43 @@ Also its fine to keep one liner styles in view
 [rn/view {:style {:flex 1 :padding-top 5}}]
 ```
 
+### Abstract styles and use the vector syntax
```

Comment:
It helps with readability if we separate the headline from the paragraph with a blank line, as well as a blank line separating the backticks/code sections.

Reply:
Added!